domain/src/main/java/camp/nextstep/edu/domain/ExpressionParser.kt

```diff
@@ -0,0 +1,7 @@
+package camp.nextstep.edu.domain
+
+class ExpressionParser {
+    companion object {
```

Comment:
특별한 상태를 가지지 않는 객체의 경우 특별히 객체로 두지 않아도 괜찮습니다.
```suggestion
object ExpressionParser {
```